lib/fxpay/pay.js

```diff
@@ -79,7 +84,8 @@
   };
 
 
-  function processWebPayment(paymentWindow, payJwt, callback) {
+  function processWebPayment(paymentWindow, managePaymentWindow, payJwt,
```

Comment:
Wondering if this is getting to the point where passing an object of args would be clearer?